cmake/developer_package/frontends/frontends.cmake

```diff
@@ -319,7 +319,7 @@ macro(ov_add_frontend)
             if(OV_FRONTEND_LINKABLE_FRONTEND)
                 set(export_set EXPORT OpenVINOTargets)
                 set(archive_dest ARCHIVE DESTINATION ${OV_CPACK_ARCHIVEDIR} COMPONENT ${dev_component} ${OV_CPACK_COMP_CORE_DEV_EXCLUDE_ALL})
-                set(namelink NAMELINK_COMPONENT ${dev_component})
+                # set(namelink NAMELINK_COMPONENT ${dev_component})
```

Comment:
TODO: core_dev target installs symbolic links and wheel, need to exclude them from the wheel

Reply:
I think we can define new component name for symlink:
- for wheel we can drop this component via exclude_from_all and other filtering
- for other cpack generator, we can redefine to core_dev like here https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/packaging/common-libraries.cmake#L73